index.html

```diff
@@ -190,6 +190,7 @@ <h2>Useful links</h2>
                         <li><a href="https://www.thegenealogist.co.uk/" target="_blank" title="Link opens in a new window">The Genealogist</a></li>
                         <li><a href="https://www.familysearch.org/en/" target="_blank" title="Link opens in a new window">FamilySearch</a></li>
                         <li><a href="https://lst.gale.com/mss/start.do?p=SPOL&u=pro&authCount=1" target="_blank" title="Link opens in a new window">State Papers Online</a></li>
+                          <li><a href="https://www.nationalarchives.gov.uk/webarchive/" target="_blank" title="Link opens in a new window">UK Government Web Archive</a></li> 
```

Comment:
Ideally, these should all be in the format:

```suggestion
                          <li><a href="https://www.nationalarchives.gov.uk/webarchive/" target="_blank" title="UK Government Web Archive (opens in a new window)">UK Government Web Archive</a></li> 
```

Do we just roll with what we have for now?

Reply:
Yes, I'd say let's keep it as is - the Reader Screens are on site only, so only on a dozen screens at most.